Dockerfiles/mlpack-dependencies-debian/Dockerfile

```diff
@@ -0,0 +1,58 @@
+#Using debian's latest image as base-image
+FROM debian:latest
+
+RUN apt-get update  && apt-get install -y aptitude && apt-get purge -y \
+    $(aptitude search '~i!~M!~prequired!~pimportant!~R~prequired! \
+    ~R~R~prequired!~R~pimportant!~R~R~pimportant!busybox!grub!initramfs-tools' \
+    | awk '{print $2}') && apt-get purge -y aptitude && \
+    apt-get autoremove -y && apt-get clean && rm -rf /usr/share/man/?? && \
+    rm -rf /usr/share/man/??_*
```

Comment:
Did you check if the aptitude installation actually does make the container smaller?  I was playing with another container yesterday and tried adding this line, but it actually ended up making the container larger!  I think in that situation the image was already so minimal that there was nothing for aptitude to remove.  But I don't know if that's true here.